mmv1/third_party/terraform/website/docs/d/backup_dr_fetch_backups.html.markdown

```diff
@@ -0,0 +1,25 @@
+---
+subcategory: "Backup and DR Service"
+description: |-
+  Get information about Backup and DR backups.
+---
+
+# google_backup_dr_fetch_backups
+
+A list of Backup and DR backups.
+
+~> **Warning:** This resource is in beta, and should be used with the terraform-provider-google-beta provider.
+See [Provider Versions](https://terraform.io/docs/providers/google/guides/provider_versions.html) for more details on beta resources.
+
+## Example Usage
```

Comment:
This doc needs to have an argument reference to tell users what they need to specify and what they can read from the resource

Reply:
See https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/d/backup_dr_data_source.html.markdown#argument-reference